core/tests/protractor_utils/AdminPage.js

```diff
@@ -272,6 +287,50 @@ var AdminPage = function() {
       expect(name).toEqual(expectedUsernamesArray[j]);
     }
   };
+
+  this.addReviewer = async function(name, newRole) {
+    await waitFor.elementToBeClickable(
+      adminRolesTab, 'Admin Roles tab is not clickable');
+    await adminRolesTab.click();
+
+    // Change values for "add reviewer for community" form, and submit it.
+    await waitFor.visibilityOf(
+      addReviewerName, 'Add reviewer name is not visible');
+    await addReviewerName.sendKeys(name);
+
+    var roleOption = selectReviewerRole.element(
+      by.cssContainingText('option', newRole));
+    await waitFor.visibilityOf(
+      roleOption, 'Reviewer role option is not visible');
+    await roleOption.click();
+
+    await addReviewerFormSubmit.click();
+    await waitFor.visibilityOf(
+      statusMessage, 'Confirmation message not visible');
+    await waitFor.textToBePresentInElement(
+      statusMessage, 'Successfully added ',
+      'Could not add reviewer successfully');
+  };
+
+  this.getReviewersAssignedToRole = async function(role) {
```

Comment:
`get` makes it sound like this function returns the reviewers. Maybe `show` or `view` instead?

Reply:
Sounds good.